lazer/contracts/evm/src/PythLazer.sol

```diff
@@ -62,7 +63,13 @@ contract PythLazer is OwnableUpgradeable, UUPSUpgradeable {
 
     function verifyUpdate(
         bytes calldata update
-    ) external view returns (bytes calldata payload, address signer) {
+    ) external payable returns (bytes calldata payload, address signer) {
+        // Require fee and refund excess
+        require(msg.value >= verification_fee, "Insufficient fee provided");
+        if (msg.value > verification_fee) {
+            payable(msg.sender).transfer(msg.value - verification_fee);
+        }
```

Comment:
I'm a little confused, what happens when msg.value == verification_fee?

Reply:
Then we wouldn't make a transfer call, since the user provided exactly the amount of fee required. That's my understanding anyway.

Reply:
oh i saw this is the refund.